app/views/static/cloud_subnet/cloud-subnet-form.html.haml

```diff
@@ -31,7 +31,7 @@
                             'maxlength' => 128,
                             'ng-model'  => 'vm.cloudSubnetModel.ext_management_system.name'}
     .form-group{'ng-class' => '{"has-error": angularForm.cloud_tenant_id.$invalid}',
-                'ng-if' => 'vm.cloudSubnetModel.ems_id'}
+                'ng-if' => 'vm.cloudSubnetModel.ems_id && vm.available_tenants.length'}
```

Comment:
At the risk of showing that I have no idea how the UI haml code works...@skateman isn't `.length => 0` still truthy?

```
>> ems.id
=> 1
>> puts "true" if ems.id && [].length
true
```

Would `vm.available_tenants.any?` be more accurate

Reply:
Actually in JS `true && 0` returns `0` which evaluates to `false` in an `if` statement.

Reply:
In that case LGTM